File: Lecture003 If-Else, While loop & Lots of Patterns/sumOfN.java
import java.util.Scanner;

class SumOfN {
    public static void main(String[] args) {
        int n;
        int sum = 0;

        Scanner sc = new Scanner(System.in);
        System.out.print("Enter the value of n: ");

        n = sc.nextInt();

        sc.close();

        for (int i = 1; i <= n; i++) {
            sum += i; // shorthand notation of sum = sum + i
        }

        System.out.println("Value of sum is: " + sum);
    }
}

File: Lecture003 If-Else, While loop & Lots of Patterns/sumOfN.js
// If you want to take JavaScript input into the browser
/*
const n = parseInt(prompt("Enter a positive integer:"));
let i = 1;
let sum = 0;

while (i <= n) {
  sum = sum + i;
  i = i + 1;
}

console.log(`value of sum is ${sum}.`);
*/

// If you want to take input on the JavaScript command line
const readline = require("readline").createInterface({
  input: process.stdin,
  output: process.stdout,
});

readline.question("Enter a positive integer: ", (input) => {
  const n = parseInt(input);
  let i = 1;
  let sum = 0;

  while (i <= n) {
    sum = sum + i;
    i = i + 1;
  }

  console.log(`value of sum is ${sum}.`);

  readline.close();
});
